dataland-frontend/src/components/forms/parts/elements/derived/ProductionSiteFormElement.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="form-field">
+    <InputTextFormField
+      name="nameOfProductionSite"
+      info="Please state the name of the production site."
+      display-name="Production Site"
+      validation="required"
+    />
+  </div>
+
+  <div class="form-field">
+    <AddressFormField
+      name="addressOfProductionSite"
+      info="Please state the address of the production site."
+      display-name="Production Site Address"
+      validation="required"
+    />
+  </div>
+
+  <div class="form-field">
+    <div class="flex justify-content-between">
+      <UploadFormHeader name="Lists of Goods or Services" explanation="Provide List of Goods or Services" />
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/47553592/d98608d3-57d6-4a1c-9d59-5a39df14de3d)

If i click "+" the text box should be cleared
No duplicates

Reply:
ne PR for this